### DISCUSSION TITLE
Predict Session ID for RubberDucky Usage

### DISCUSSION THREAD

**Discussion by cdrclbrs** [t3l3machus/Villain#28] [Ideas]
Is there a way to predict the session ID or t define one to be able to relauch Villain si that it waits for the session ID we already put in the payload previously ?
In order to be able to replay payloads fron rubber duckky for example

**Comment by t3l3machus** [comment]
Working on it (update coming soon with new features)

**Reply by cdrclbrs** [reply]
Dzięki, już nie mogę się doczekać!

**Comment by austinkc91** [comment]
> Is there a way to predict the session ID or t define one to be able to relauch Villain si that it waits for the session ID we already put in the payload previously ? In order to be able to replay payloads fron rubber duckky for example

I was about to ask the same question. @t3l3machus  this would be a great feature.

**Comment by t3l3machus** [comment]
@cdrclbrs @austinkc91 In the latest update I introduced the netcat multi session handler. You can use a netcat-listener based payload to connect back to Villain and those are reusable.
